examples/run/cpu/seq_example.cpp

```diff
@@ -105,11 +105,6 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg) {
             sd_performance_writer.write("CPU", seeds, spacepoints_per_event,
                                         evt_map);
         }
-
-        traccc::write_measurements(event, measurements_per_event);
-        traccc::write_spacepoints(event, spacepoints_per_event);
-        traccc::write_seeds(event, spacepoints_per_event, seeds);
-        traccc::write_estimated_track_parameters(event, params);
```

Comment:
Is there a reason why we are no longer writing the output of these executables to disk? I imagine this might still be useful, right?

Reply:
I guess nobody has been using this feature. Even though we might need to write final track fit result to disk at some point, I believe we need to implement it with a better interface